es/features/graphql.md

```diff
@@ -1,24 +1,25 @@
 # Documentación de la API
 
-* [Características](#características)
+* [Características](#caracteristicas)
 * [GraphQL](#graphql)
 * [Haciendo peticiones a la API](#haciendo-peticiones-a-la-api)
   * [Clientes soportados](#clientes-soportados)
     * [GraphiQL](#graphiql)
     * [Postman](#postman)
-    * [Librerías HTTP](#librerías-http)
-* [Información disponible](#información-disponible)
+    * [Librerías HTTP](#librerias-http)
+* [Información disponible](#informacion-disponible)
 * [Ejemplos de consultas](#ejemplos-de-consultas)
-  * [Recuperar un único elemento de una colección](#recuperar-un-único-elemento-de-una-colección)
-  * [Recuperar una colección completa](#recuperar-una-colección-completa)
-    * [Paginación](#paginación)
-  * [Acceder a varios recursos en una única petición](#acceder-a-varios-recursos-en-una-única-petición)
+  * [Recuperar un único elemento de una colección](#recuperar-un-unico-elemento-de-una-coleccion)
+  * [Recuperar una colección completa](#recuperar-una-coleccion-completa)
+    * [Paginación](#paginacion)
+  * [Acceder a varios recursos en una única petición](#acceder-a-varios-recursos-en-una-unica-peticion)
 * [Limitaciones de seguridad](#limitaciones-de-seguridad)
   * [Ejemplo de consulta demasiado profunda](#ejemplo-de-consulta-demasiado-profunda)
   * [Ejemplo de consulta demasiado compleja](#ejemplo-de-consulta-demasiado-compleja)
-* [Ejemplos de código](#ejemplos-de-código)
+* [Ejemplos de código](#ejemplos-de-codigo)
 
-## Características
+<!-- ## Características -->
```

Comment:
Do you think it's necessary to keep the old code in a comment? :thinking: If I were reading the code, it would be a bit confusing and I'd spend some time trying to understand why the comment is there.

Same thing applies to the other changes in this pull request.

I'd vote for removing the comments, but if there's a reason why we should keep them, let me know :wink:.

Reply:
I left the comments because I thought it helped to understand the change… but if we think it might be confusing I'll remove them (e0d1c88)
Thank you for reviewing!